backend/src/modules/users/presentation/rest-api/UserProfileRouter.ts

```diff
@@ -0,0 +1,71 @@
+import { CommandPublisher } from '../../../../shared/core/application/command/CommandBus';
+import { DomainEventPublisher } from '../../../../shared/core/application/event/DomainEventBus';
+import { QueryPublisher } from '../../../../shared/core/application/query/QueryBus';
+import express, { Request, Response } from 'express';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import { PostUserProfileRequestBody } from './request/PostUserProfileRequestBody';
+import {
+  FindUserProfileByEmail,
+  FindUserProfileByEmailResult,
+} from '../../core/application/query/FindUserProfileByEmail';
+import { StatusCodes } from 'http-status-codes';
+import { CreateUserProfile } from '../../core/application/command/CreateUserProfile';
+
+export function userProfileRouter(
+  commandPublisher: CommandPublisher,
+  eventPublisher: DomainEventPublisher,
+  queryPublisher: QueryPublisher,
+): express.Router {
+  const signIn = async (request: Request, response: Response) => {
```

Comment:
Co ma byc tym userProfile? Chyba pomieszales jakies koncepty. displayName nie ma zadnego polaczenia z password. Wiec to raczej nie ten sam modul. Nie ma zadnych zasad miedzy nimi. Czy password musi sie zaczynac np. an peirwsaa litere desplayName :P ? To powinien byc modul do uwierzytelnienia. A drugie to moze byc taki zwykly CRUD z danymi usera jak np. displayName czy jakis opis itp.